scripts/RyzenAdj.spec

```diff
@@ -0,0 +1,33 @@
+Name: RyzenAdj
+Version: 0.11.1
```

Comment:
Who does maintain this version number? There is a risk that we forget to change it during the release.

Reply:
That is a good question. As COPR integrates with Github hooks, there should be a variable for it. I will dig into it.

Reply:
> [That's it, the packages builds correctly on COPR](https://copr.fedorainfracloud.org/coprs/augier/RyzenAdj/package/RyzenAdj/).
> 
> So, basically, this is how you should maintain the `.spec` file:
> 
>     * change the `Version` variable to the next version release before tagging a new git version,
> 
>     * also add a new entry at the top of the `%changelog` section summarizing the changes in the `.spec` file, if any; if none, just add "_Update to version x.y.z_".
> 
> 
> A new entry in the `%changelog` section should look like this:
> 
> ```
> * Day-of-Week Month Day Year Name Surname <email> - Version-Release
> - list of changes
> ```
> 
> If the build process doesn't change, this spec file should be stable over time.

Don't know if it is possible to use the 'tito' tool to change the version to "git describe" as of the github actions in my PR. I.E will change the version of each commit. 

You should be able to build the package inside a [docker container](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container) in the same github action flow. 

As for the change log. It is more of importance to track per distro changes, not upstream changes. I.E if the distro specific stuff is upstream we don't need to keep the change log as strict as a normal Debian/rpm package.

Regarding the packaging in this PR. Maybe we could follow the same "app", lib and lib-dev ? Think it should be quite easy in .spec. Probably should rebase this change on top of mine to get the "install target" from cmake correct.